src/main/java/edu/ie3/datamodel/models/input/container/InputContainer.java

```diff
@@ -15,6 +15,11 @@
   /** @return unmodifiable List of all entities */
   List<T> allEntitiesAsList();
 
+  /** Returns number of holded elements in the container. */
+  default int numberOfElements() {
+    return allEntitiesAsList().size();
+  }
+
```

Comment:
This method is never used. Maybe we don't need it. 

Also "holds"

Reply:
Oh yes, in this implementation it isn't necessary. I'll remove it.
As a remark: In failing groovy tests all list elements will be displayed. Then the error message is very big and messy. This method was a way to avoid that.